<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWidget</class>
 <widget class="QWidget" name="StatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史使用统计</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboBox_view_mode">
       <property name="toolTip">
        <string>选择视图方式(注：日视图最多显示7天的内容)</string>
       </property>
       <item>
        <property name="text">
         <string>日视图</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周视图</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>月视图</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_begin">
       <property name="toolTip">
        <string>请选择开始日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_end">
       <property name="toolTip">
        <string>请选择结束日期</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="GraphicLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
